turbo.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["dist/**"]
+    },
+    "lint": {},
```

Comment:
Since the project is on top of typescript, `lint` should depend on `^build`. Otherwise the linter might have a hard time trying to resolve types from package dependencies that are not built.

Reply:
Done.